### DISCUSSION TITLE
Repository

### DISCUSSION THREAD

**Discussion by Caaasperrr** [community/community#62607] [Repositories]
### Select Topic Area

Question

### Body

How do i open a repository?

**Comment by FelixHennerich** [comment]
Hi, 
go to github.com then navigate to your profile picture in the top right. Go to "your repository". At last u have to press the green new button. 
I hope it helps you :))

**Reply by JiveyGuy** [reply]
Hey Felix, 

Thank you for taking the time to answer this. On other programming QA sites that rhyme with pack under stow people get very upset at these low-effort (Google-able) questions that show the author didn't attempt to find any info themselves. 

However, responding with positivity and a helpful direct answer will make the GitHub community a great place for the future. 

Thank you!

**Comment by queenofcorgis** [comment]
@Caaasperrr @FelixHennerich GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
